attachments/models.py

```diff
@@ -242,7 +240,14 @@ def upload_forms(self):
         for upload in self.uploads.all():
             error_msg = self.validate_attachment(upload)
             if not error_msg:
-                yield None, upload, self._forms.get(upload, PropertyForm(instance=upload, editable_only=False))
+                property_form = PropertyForm(initial=self.data, instance=upload, editable_only=False)
```

Comment:
Is the initial even necessary? Seems like the PropertyForm pulls initial field data directly from `instance.session.data`.

Reply:
Ah yea, you're right. It's not needed, I'll remove it